build.sbt

```diff
@@ -29,11 +29,11 @@ libraryDependencies ++= Seq(
   "com.typesafe.slick" %% "slick" % "3.0.0",
   "com.zaxxer" % "HikariCP" % "2.3.3",
   "org.hsqldb" % "hsqldb" % "2.3.2",
-  "mysql" % "mysql-connector-java" % "5.1.35",
   //---------- Test libraries -------------------//
   "io.spray" %% "spray-testkit" % sprayV % Test,
   "org.scalatest" %% "scalatest" % "2.2.5" % Test,
   "com.typesafe.akka" %% "akka-testkit" % akkaV % Test,
+  "mysql" % "mysql-connector-java" % "5.1.35",
```

Comment:
Should this have relocated to the "Test libraries" neighborhood (and not been scoped to `Test`)?